components/AirdropLogin.vue

```diff
@@ -108,22 +109,39 @@
   </div>
 </template>
 <script lang="ts">
-import { Vue, Component, Prop } from 'vue-property-decorator'
-import { CONNECT_WALLET_TYPES } from '~/constant'
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator'
+import {
+  CONNECT_WALLET_TYPES,
+  COSMOS_ADDRESS_REGEX,
+  OSMO_ADDRESS_REGEX,
+} from '~/constant'
 
 @Component
 export default class AirdropLogin extends Vue {
   @Prop(Boolean) readonly isAirdropStarted: boolean | undefined
 
   inputAddress: string = ''
+  errorMessage: string = ''
 
   // eslint-disable-next-line class-methods-use-this
   get connectWalletTypes() {
     return CONNECT_WALLET_TYPES
   }
 
+  @Watch('inputAddress')
+  onValueChange() {
+    this.errorMessage = ''
+  }
+
```

Comment:
I think you can use `@input` on `textfield` instead of using watch, avoid watch if possible